db/seeds.rb

```diff
@@ -284,40 +284,55 @@
 Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id)
 Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id)
 Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_printer.id)
 Request.create!(user_id: user_seeker_4.id, category_id: category_camera.id)
 Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id)
 Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, status: "Completed")
 
 #bookings
-Booking.create!(charity_id: charity2.id,
-                user_id: user_seeker_1.id,
-                product_id: product_dell_laptop.id,
+Booking.create!(charity_id: charity2.id, 
+                user_id: user_seeker_2.id, 
+                product_id: product_dell_laptop.id, 
+                status: "Accepted"
+                )
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_1.id, 
+                product_id: product_macbook.id, 
                 status: "Declined"
                 )
-Booking.create!(charity_id: charity2.id,
-                user_id: user_seeker_2.id,
-                product_id: product_macbook.id,
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_8.id, 
+                product_id: product_mobile.id, 
```

Comment:
Hi @GayatriArora, it seems like this seeker has not made a request for a mobile?